internal/daemon/daemon.go

```diff
@@ -380,7 +391,7 @@ func (d *Daemon) StartAPI(bootstrap bool, initConfig map[string]string, newConfi
 			return err
 		}
 
-		return d.hooks.OnBootstrap(d.State(), initConfig)
+		return d.hooks.PostBootstrap(d.State(), initConfig)
```

Comment:
I am wondering if we should handle the `error` returned from `PostBootstrap` explicitly meaning adding the description as for `PreBootstrap` above. Something like `Failed to run post-bootstrap hook before starting the API: %w`